auth_oidc/models/res_users.py

```diff
@@ -30,7 +30,6 @@ def _auth_oauth_get_tokens_auth_code_flow(self, oauth_provider, params):
         response = requests.post(
             oauth_provider.token_endpoint,
             data=dict(
-                client_id=oauth_provider.client_id,
```

Comment:
this is necessary for [okta](https://okta.com), as it validates more strictly - also according to the [specs](https://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication) it's wrong to pass this when the authentication method is `client_secret_basic` (the default, we don't do anything else)